mitodl/ocw-studio
# feat: show short_id in course list select

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #393 

#### What's this PR do?

This PR shows `short_id` in the label when selecting a course for a course list.

#### How should this be manually tested?
- Navigate to your local setup of cow-studio
- Checkout `hussaintaj/393-short-id`
- Open studio and head to `/sites`
- Open your site `ocw-www`
- Add a new course list or edit an existing one
- Open the courses dropdown and observe the title. **Expectation:** the format is `title (short_id)`
- Search courses and observe the title of the result. **Expectation:** the format is `title (short_id)`
- Select course(s) and add them to the list
- Save the course list and verify that the course list is successfully saved.
- Publish the course and check the generated HTML under `content/course-lists/`. **Expectation**: You should not see any unnecessary short_ids.

#### Screenshots (if appropriate)

<img width="529" alt="Screenshot 2023-03-29 at 3 22 23 PM" src="https://user-images.githubusercontent.com/71316217/228504955-ed98b9ec-44f1-4be1-a2f9-1b5ba25d2d4f.png">
<img width="525" alt="Screenshot 2023-03-29 at 3 22 14 PM" src="https://user-images.githubusercontent.com/71316217/228504968-3b1f8ebd-eb99-4570-8938-da28560a96e3.png">
<img width="540" alt="Screenshot 2023-03-29 at 3 22 07 PM" src="https://user-images.githubusercontent.com/71316217/228504977-e0070f58-34fc-4368-ad07-7d16f8aee373.png">
